xrdsst/core/validator.py

```diff
@@ -0,0 +1,228 @@
+import copy
```

Comment:
What is the point of these:

def validate_config_timestamp_init(ss_config, operation, errors):
    # Timestamping service initialization uses only information from central server and security server
    # and has no specific configuration file section.
    return True
    
def validate_config_cert_register(ss_config, operation, errors):
    # AUTH cert registration has no specific configuration file section / keys.
    return True


def validate_config_cert_activate(ss_config, operation, errors):
    # AUTH cert activation has no specific configuration file section / keys.
    return True

Reply:
These are referred as first part of the change description and implement the interface methods in one place.